src/pages/markets/components/helper/_symbol.js

```diff
@@ -3,24 +3,14 @@ import PropTypes from 'prop-types'
 import { SymbolContainer } from '../../static/style/_markets-style'
 import { Text } from 'components/elements'
 
-//TODO: refactor this component to always use instruments_type
-const Symbol = ({ instruments_type, src, text }) => (
+const Symbol = ({ instruments_type }) => (
     <React.Fragment>
-        {instruments_type ? (
-            <React.Fragment>
-                {instruments_type.map((symbol, index) => (
-                    <SymbolContainer key={index}>
-                        <img src={symbol.src} />
-                        <Text>{symbol.text}</Text>
-                    </SymbolContainer>
-                ))}
-            </React.Fragment>
-        ) : (
-            <SymbolContainer>
-                <img src={src} />
-                <Text>{text}</Text>
+        {instruments_type.map((symbol, index) => (
+            <SymbolContainer key={index}>
+                <img src={symbol.src} />
```

Comment:
```suggestion
                <img src={symbol.src} alt="" />
```
We can leave an empty alt here as a best practice for visual icons.